### PR TITLE
Misc improvements

### PR DESCRIPTION
**Pull request by fviernau**
Fix-up to if conditions and improve logging around the `config.yml`.

### CODE REVIEW

File: templates/ort-scan.yml [hunk 1]
```diff
@@ -158,10 +158,13 @@
         && export ORT_CONFIG_VCS_URL=$(git config remote.origin.url) \
         && export ORT_CONFIG_VCS_REVISION=$(git rev-parse HEAD)      
       cd -
+     
+      echo -e "Using ORT config revision ${ORT_CONFIG_VCS_REVISION}."
 
     # Generate ORT global configuration if not found and PostgreSQL is used.
     - |
       if [[ (! -z "${ORT_CONFIG_PATH}/config.yml") && (-n "${POSTGRES_URL}") ]]; then
+      echo -e "\e[1;33m Generating a 'config.yml' with a postgres storage..."
```
Comment (tsteenbe) [inline templates/ort-scan.yml]: Suggest to change "postgres" to "PostgreSQL"

File: templates/ort-scan.yml [hunk 2]
```diff
@@ -231,8 +235,12 @@
           - type: OrtConfig
             enabled: false
       EOF
+      else
+      echo -e "\e[1;33m Using 'config.yml' from 'ort-config' repository..."
       fi
-    
+      
+      ${ORT_CLI} config --show-active
```
Comment (tsteenbe) [inline templates/ort-scan.yml:248]: Should we always print the active ORT config - might contain sensitive data

File: templates/ort-scan.yml [hunk 3]
```diff
@@ -109,8 +109,8 @@
 
       if [ ! -n "${VCS_TYPE+x}" ] && \
         [ ! -n "${VCS_URL+x}" ] && \
-        [ ! -n "${VCS_REVISION}" ] && \
-        [ ! -n "${VCS_PATH}" ]
+        [ ! -n "${VCS_REVISION+x}" ] && \
```
Comment (tsteenbe) [inline templates/ort-scan.yml:112]: What do the `+x` do? The commit message does not explain it

Comment (fviernau) [reply templates/ort-scan.yml:112]: I've taken this from here: https://stackoverflow.com/questions/3601515/how-to-check-if-a-variable-is-set-in-bash.
Not sure if it's needed though. What do you think? Shall we just drop it? Note: The two lines above already have the "+x" from one of my preceding changes.

File: templates/ort-scan.yml (hunk 2 above)
Comment (fviernau) [reply templates/ort-scan.yml:248]: Looks like we have a mechanism for stipping it out. E.g. it does not show the password for the JDBC connection. To me it's really helpful to see the active config in recent days.

**Review by tsteenbe — APPROVED** [review]

**Pull request merged**